app/Scrambler.py

```diff
@@ -0,0 +1,64 @@
+# pylint: disable-all
+
+from random import choice, shuffle
+
+from Picture import Picture
+from Tile import Tile
+
+
+class Scrambler:
+    @classmethod
+    def scramble_rows(cls, picture: Picture) -> None:
+        """Splits an Image up into rows an rearranges them."""
+        num_tiles = 8
+        image_size = picture.image.size
+        tile_size = (image_size[0], int(image_size[1] / num_tiles))
+
+        for y in range(num_tiles):
+            x0, y0, x1, y1 = (0, tile_size[1] * y, tile_size[0], tile_size[1] * (y + 1))
+            picture.tiles[y] = Tile(picture.image.crop((x0, y0, x1, y1)), (x0, y0))
+
+        picture.tile_order = list(picture.tiles.keys())
+        shuffle(picture.tile_order)
+
+    @classmethod
+    def scramble_grid(cls, picture: Picture) -> None:
+        """Splits an Image up into tiles an rearranges them and rotates each tile randomly."""
+        num_of_tiles = 4
+        co_ordinates = [(0, 0, 512, 512), (512, 0, 1024, 512), (0, 512, 512, 1024), (512, 512, 1024, 1024)]
+
+        # Tile swap logic
+        tile_order = [0, 1, 2, 3]
+        swap_list = ["top", "left", "bottom", "right"]
+        tile_rotations = [choice([90, 180, 270]) for _ in range(num_of_tiles)]
```

Comment:
Would it make sense to have the option for a tile to not be rotated? This would mean adding `0` as an option to the list.

Reply:
Adding 0 sometimes creates a situation where there are many tiles not being rotated which causes the captcha to be really easy. Rotating every tile at least once gives a bit of complexity.